manim_slides/config.py

```diff
@@ -149,21 +149,22 @@ def slides_slice(self) -> slice:
 class PresentationConfig(BaseModel):  # type: ignore
     slides: List[SlideConfig]
     files: List[FilePath]
+    resolution: Tuple[PositiveInt, PositiveInt] = (1920, 1080)
```

Comment:
@Fairlight8 first, I added a new `resolution` field. Specifying a default value is very important for backward compatibility (i.e., what to do when no resolution was specified? Use the default!).